scipy/signal/signaltools.py

```diff
@@ -2971,13 +2971,13 @@ def decimate(x, q, n=None, ftype='iir', axis=-1, zero_phase=None):
     if ftype == 'fir':
         if n is None:
             n = 30
-        system = lti(firwin(n+1, 1. / q, window='hamming'), 1.)
+        system = dlti(firwin(n+1, 1. / q, window='hamming'), 1.)
     elif ftype == 'iir':
         if n is None:
             n = 8
-        system = lti(*cheby1(n, 0.05, 0.8 / q))
-    elif isinstance(ftype, lti):
```

Comment:
It was added in gh-5392, which hasn't landed in an actual release, so I don't think that we're technically breaking backwards compatibility. This is part of why I wanted the `dlti` systems to land in 0.18, so we didn't let this incorrect usage of `lti` through.